git-extra/vimrc

```diff
@@ -20,13 +20,13 @@ set statusline=%<%F%h%m%r\ [%{&ff}]\ (%{strftime(\"%H:%M\ %d/%m/%Y\",getftime(ex
 " Only do this part when compiled with support for autocommands.
 if has("autocmd")
     " Set UTF-8 as the default encoding for commit messages
-    autocmd BufReadPre COMMIT_EDITMSG,MERGE_MSG,git-rebase-todo setlocal fileencodings=utf-8
+    autocmd BufReadPre COMMIT_EDITMSG,MERGE_EDITMSG,MERGE_MSG,git-rebase-todo setlocal fileencodings=utf-8
```

Comment:
This comes from https://github.com/git-for-windows/build-extra/commit/f5cc31f85522b92d0ee1e7ea11de2234cd7cc37d, which references https://github.com/git-for-windows/git/issues/1207 as reason for introducing the `MERGE_EDITMSG` file name here.

As you can see easily from reading that discussion, I simply made a mistake, and `MERGE_MSG` was really intended.

Maybe take this opportunity to fix it?

Reply:
Sure, will do.